experimental/devices/tlv493d/doc.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package tlv493d implements interfacing code to the Infineon TLV493D haff effect sensor.
+//
+// Features of the device:
+// - 3-dimensional hall effect sensor, measures up to +/-130 mT magnetic flux.
```

Comment:
It won't show up nicely in godoc sadly. You'll probably want to do indent so it shows as a code block.